torch/distributed/_composable/fsdp/_fsdp_param_group.py

```diff
@@ -488,6 +495,11 @@ def _all_reduce_process_group(self) -> dist.ProcessGroup:
         assert isinstance(self.mesh_info, HSDPMeshInfo)
         return self.mesh_info.replicate_process_group
 
+    def _with_fqn(self, label: str) -> str:
```

Comment:
cc: @yf225 I specifically did not define a new context manager for this and am just using a helper to construct the string label passed into `record_function`. IIUC, this should be fine with Dynamo?